R/update_snapshot.R

```diff
@@ -164,6 +165,7 @@ update_snapshot <- function(.data, conn, db_table, timestamp, filters = NULL, me
                                 dplyr::select(.data, "checksum")) |>
       dplyr::compute() # Something has changed in dbplyr (2.2.1) that makes this compute needed.
     # Code that takes 20 secs with can be more than 30 minutes to compute without...
+    defer_db_cleanup(to_remove)
```

Comment:
Later code depends on to_remove ... let us discuss

Reply:
`defer_db_cleanup()` deletes the table once its parent (`update_snapshot`) exits. By that time, `to_remove` has been used to update the table and should no longer be needed.